src/pages/studyView/StudyViewUtils.spec.ts

```diff
@@ -216,7 +216,9 @@ describe('StudyViewUtils', () => {
                     },
                     {
                         geneQueries: [['GENE1']],
-                        molecularProfileIds: ['cancer_study_fusion'],
+                        molecularProfileIds: [
+                            'cancer_study_structural_variants',
```

Comment:
Also we should have all profiles ids we keep referring to in the study view as constants.

Reply:
I think this can't be constant. It is study specific